apps/src/templates/referenceGuides/ReferenceGuideView.jsx

```diff
@@ -1,21 +1,89 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import EnhancedSafeMarkdown from '@cdo/apps/templates/EnhancedSafeMarkdown';
+import NavigationBar from './NavigationBar';
+import color from '@cdo/apps/util/color';
+import {Link} from '@dsco_/link';
+import {organizeReferenceGuides} from '@cdo/apps/util/referenceGuideHelpers';
+import classNames from 'classnames';
 
-export default function ReferenceGuideView({referenceGuide}) {
-  return (
-    <div>
-      <h1>{referenceGuide.display_name}</h1>
-      <EnhancedSafeMarkdown markdown={referenceGuide.content} />
-    </div>
-  );
-}
+const baseUrl = window.location.href
+  .split('/')
+  .slice(0, -1)
+  .join('/');
 
 const referenceGuideShape = PropTypes.shape({
   display_name: PropTypes.string,
-  content: PropTypes.string
+  content: PropTypes.string,
+  position: PropTypes.number,
+  parent_reference_guide_key: PropTypes.string
 });
 
+const NavBarItem = ({guide, isActive}) => (
+  <div
+    style={{paddingLeft: `${guide.level * 12}px`}}
+    className={classNames({
+      'nav-link': true,
+      active: isActive
+    })}
+  >
+    <Link className="link" href={`${baseUrl}/${guide.key}`} weight="medium">
+      {guide.display_name}
+    </Link>
+  </div>
+);
+NavBarItem.propTypes = {
+  guide: referenceGuideShape.isRequired,
+  isActive: PropTypes.bool
+};
+
+export default function ReferenceGuideView({referenceGuide, referenceGuides}) {
+  let rootCategory = referenceGuide;
+  // TODO(tim): re-organize things to get rid of the concepts guide
```

Comment:
Could you explain what this TODO is tracking and if its something you are planning to do before moving to Star Labs

Reply:
Looking more closely below it looks like we have a top level concepts reference guide. Is that true? Why?

Reply:
Yeah, that's how it is coming from curriculumbuilder, but we don't actually want to display that top level guide anywhere! So I was planning to edit the import script to skip over the 'concepts' guide and just import all the children of 'concepts' as the top level guides. I'll also need to update the parent_reference_guide_key dropdown to allow setting it to `null`.

Reply:
Ok great. Thanks for the update! So that work is happening and this will be updated later to match?